Vagrantfile

```
@@ -77,6 +78,19 @@ def mount_folders(config, mount_options)
 end
 
 Vagrant.configure(2) do |config|
+  # Optional plugin for reading from env file
+  if Vagrant.has_plugin?('vagrant-env')
+    config.env.enable
+  end
+
+  # Default box
+  config.vm.box = base_boxes.default
+
+  # Custom box
+  if ENV.has_key?('VAGRANT_BOX')
+    config.vm.box = ENV['VAGRANT_BOX']
+  end
```

The more idiomatic way to write this would be:

```suggestion
  config.vm.box = ENV.fetch('VAGRANT_BOX', base_boxes.default)
```

I would also drop the `# Default box` and `# Custom box` comments. They just give a "what" the line does, and the line itself is simple enough to not warrant the comment.